### ISSUE TITLE
Do we need to create our own AMIs, or are these available on Amazon somewhere?

### ISSUE THREAD

**Issue by charford** [RetailMeNot/SeleniumGridScaler#5]
In the .properties example file under the folder resources, there are some ami IDs. I tried searching for these on AWS, but can't find them. Do I need to create my own, or are these available somewhere?

**Comment by mhardin** [comment]
Hey there.  We were going to provide the AMI's, but we have not made them public yet as no one has wanted to try the plugin out yet.  Is this something you would be interested in?

**Comment by charford** [comment]
Yes. This is something I have been considering doing myself, but given that
you guys already got this working, I wanted to try using your version
instead. Currently I am using sauce labs but am looking at alternatives to
run in house or on ec2. Making the ami's public would be awesome!

**Comment by mhardin** [comment]
Can you shoot me an email at mhardin.github@gmail.com with your AWS account number and I can share the AMI with you?

Thanks.

**Comment by arunlrk** [comment]
Is it possible for you to give access to your Windows AMIs? We are using Selenium automation at scale and it would be really helpful to use what you already have.

**Comment by mhardin** [comment]
Unfortunately since I no longer work at the company where the Windows image was maintained, I no longer have access to it.  Your options are to create a Windows instance yourself, and then duplicate the startup logic found on the linux node at ~/grid/grid_start_node.sh to the windows image.

Hope this helps.
